bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/variables.json

```diff
@@ -0,0 +1,189 @@
+{

```

Comment:
That's a bit emberrassing, but I couldn't figure out how to reference the relative path to another folder within the JAR file when reading the file's contents... So I opted for the easy way 😄

Reply:
If you keep the same directoy path in the resources directory it will work the same as when put in the java directory. The compilation process compiles everything to the same directory anyway. So please do move this file to `src/main/resources/org/openhab/binding/modbus/helioseasycontrols/internal`.

Reply:
Thanks for the hint, works like a charm 👌